src/main/java/sql/Connector.java

```diff
@@ -0,0 +1,28 @@
+package sql;
+
+import org.apache.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class Connector {
```

Comment:
As this class is util, make it final and create a private constructor to prevent creation object of this class.